lib/tokyo_api/identity.rb

```diff
@@ -8,22 +8,41 @@ def full_user(id)
       client.get_request("#{normalized_base_path}full_user/#{url_escape(id)}").body
     end
 
-    def tokyo_identity_user_path(id, required_fields: nil)
+    def tokyo_identity_user_path(id, with_subscription_status: false, required_fields: nil, opt_in_public_ids: nil, minimum_consent_level: nil)
       path = "/#{normalized_base_path}user/#{url_escape(id)}"
 
       unless required_fields.nil?
         path << "?#{required_fields_param(required_fields)}"
       end
+
+      if with_subscription_status
+        path << '?with_subscription_status=true'
```

Comment:
If `required_fields` is present, I think this will create an invalid URL with a second `?` in it instead of a `&`. Is there a URL-building library we can use here?

Reply:
✅